netmaster/main.go

```diff
@@ -209,6 +212,9 @@ func (d *daemon) ListenAndServe() {
 		})
 	}
 
+	// proxy Handler
+	router.PathPrefix("/proxy/").HandlerFunc(proxyHandler)
```

Comment:
- This is a hack to access the overlay networks in vagrant environment. Thats how Vipin demo'd accessing redid/web application from the browser on his laptop.
- Yes, we'll create a separate service routing proxy like kube-proxy